docs/standards/smart-contracts/utils/lsp10-received-vaults-utils.md

```diff
@@ -0,0 +1,99 @@
+---
+title: LSP10ReceivedVaultsUtils
+sidebar_position: 4
+---
+
+# LSP10Utils
+
+:::info
+
+[`LSP10Utils.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP10ReceivedVaults/LSP10Utils.sol)
+
+:::
+
+This library should be used to generate ERC725Y data keys and values for transferring vaults (LSP9).
+
+## Functions
+
+### generateReceivedVaultKeys
+
+```solidity
+function generateReceivedVaultKeys(
+    address receiver,
+    address vault,
+    bytes32 vaultMapKey,
+    bytes4 interfaceID
+) internal view returns (bytes32[] memory keys, bytes[] memory values);
+```
+
+Generating the data keys/values to be set on the receiver address after receiving vaults.
+
+#### Parameters:
+
+| Name        | Type    | Description                                                                                                 |
+| :---------- | :------ | :---------------------------------------------------------------------------------------------------------- |
+| receiver    | address | The address receiving the vault and where the Keys should be added.                                         |
+| vault       | address | The address of the vault being received.                                                                    |
+| vaultMapKey | bytes32 | The map key of the vault being received containing the interfaceId of the vault and the index in the array. |
```

Comment:
I think it would make more sense to describe:

**what is the vaulMapKey** (=how it is constructed, as the LSP10Vault Map prefix + the `vault` address) ✅ 

instead of **what is contained inside it**. 

It would help better for developers to understand what they need to pass as parameter.